content/panorama/scripts/custom_game/hero_selection.js

```diff
@@ -321,21 +324,16 @@ function onPlayerStatChange (table, key, data) {
   } else if (key === 'time' && data != null) {
     // $.Msg(data);
     if (data.mode === 'STRATEGY') {
-      if (!iscm) {
-        $.Msg('FinishPickings');
-        ReturnChatWindow();
-        SetupTopBar();
-      }
       FindDotaHudElement('TimeLeft').text = 'VS';
       FindDotaHudElement('GameMode').text = $.Localize(data['mode']);
       GoToStrategy();
     } else if (data['time'] > -1) {
       FindDotaHudElement('TimeLeft').text = data['time'];
       FindDotaHudElement('GameMode').text = $.Localize(data['mode']);
     } else {
-      // CM Hides the chat on last pick before selecting plyer hero
-      // ARDM don't have pick screen
-      if (currentMap === 'oaa') {
+      // CM Hides the chat on last pick, before selecting plyer hero
+      // ARDM don't have pick screen chat
+      if (currentMap === 'oaa' || currentMap === 'oaa_10v10') {
```

Comment:
Yeah, this was the thing i realized needed to be done after thinking back on the changes I made... Nice find.